src/ResourceManager/Network/Commands.Network/help/Add-AzureRmNetworkInterfaceIpConfig.md

```diff
@@ -343,15 +339,15 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ### Microsoft.Azure.Commands.Network.Models.PSNetworkInterface
 Parameters: NetworkInterface (ByValue)
 
-### System.Collections.Generic.List`1[[System.String, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]]
+### System.String[], mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
```

Comment:
@EvgenyAgafonchikov nit: I think having just `System.String[]` should be fine here (and in all the spots where we went from list to array in the `INPUTS` and `OUTPUTS` sections -- I think the trailing content with the versions is only used in lists. We can clean this up later since this shouldn't block the PR from going in.